QuickSwiftCheck.xcodeproj/project.pbxproj

```diff
@@ -280,13 +280,16 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 3AD546101F17A0D50000CCCA /* Common-Debug.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTING_SEARCH_PATHS = YES;
```

Comment:
QuickSwiftCheck manages build settings exclusively through xcconfig files. Please remove this setting from here and add it at the end of [QuickSwiftCheck/Configurations/Common/Common-Base.xcconfig](https://github.com/akashivskyy/QuickSwiftCheck/blob/develop/QuickSwiftCheck/Configurations/Common/Common-Base.xcconfig):

```none
// MARK: Custom

ENABLE_TESTING_SEARCH_PATHS = YES;
```